travis/test_pylint

```diff
@@ -9,12 +9,54 @@ import os
 
 import run_pylint
 
+from getaddons import get_modules_changed
 
 pylint_rcfile = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     'cfg',
     "travis_run_pylint.cfg")
-result = run_pylint.main([
+count_errors = run_pylint.main([
     "--config-file=" + pylint_rcfile,
 ], standalone_mode=False)
-exit(result)
+
+
+pylint_rcfile_pr = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'cfg',
+    "travis_run_pylint_pr.cfg")
+
+is_pull_request = os.environ.get(
+    'TRAVIS_PULL_REQUEST', 'false') != 'false'
```

Comment:
TRAVIS_PULL_REQUEST is a [environment variable of travis](http://docs.travis-ci.com/user/environment-variables/#Convenience-Variables)
![environment variables - travis ci 2015-08-21 09-41-33](https://cloud.githubusercontent.com/assets/6644187/9410958/d3e63378-47e8-11e5-85ce-b64515c24c43.png)
This is needed here because we need to know if is a PR to verify new conventions just in PR cases to detect a diff with base branch `TRAVIS_BRANCH`.
![environment variables - travis ci 2015-08-21 09-43-57](https://cloud.githubusercontent.com/assets/6644187/9411021/29e8fea4-47e9-11e5-82e1-25269c279a18.png)

Now we can get git diff of `TRAVIS_BUILD_DIR`
![environment variables - travis ci 2015-08-21 09-45-24](https://cloud.githubusercontent.com/assets/6644187/9411044/5d114688-47e9-11e5-8036-a53b2d53e20d.png)

With next command if is PR:
```python
  git --git_dir=${TRAVIS_BUILD_DIR}/.git diff-index origin/${TRAVIS_BRANCH}
```

Now, we have all items changed in a PR we need verify if is a folder and a module to add to paths-to-verify in pylint with new conventions (pylint conf pr file).